Fabulous.CodeGen/src/Fabulous.CodeGen/AssemblyReader/Reader.fs

```diff
@@ -111,15 +111,15 @@ module Reader =
         
         { FullName = tdef.FullName
           AssemblyName = tdef.Module.Assembly.Name.Name
-          CanBeInstantiated = not tdef.IsAbstract && ctor.IsSome && ctor.Value.Parameters.Count = 0
+          CanBeInstantiated = not tdef.IsAbstract && ctor.IsSome // && ctor.Value.Parameters.Count = 0
           InheritanceHierarchy = Resolver.getHierarchyForType baseTypeName tdef 
           Events = readEventsFromType convertTypeName tdef
           AttachedProperties = readAttachedPropertiesFromType convertTypeName tryGetStringRepresentationOfDefaultValue tryGetProperty propertyBaseType tdef
           Properties = readPropertiesFromType convertTypeName tryGetStringRepresentationOfDefaultValue tryGetProperty tdef }
         
     let readAssemblies
             (loadAllAssembliesByReflection: seq<string> -> Assembly array)
-            (tryGetAttachedPropertyByReflection: Assembly array -> string * string -> Models.ReflectionAttachedProperty option)
+            (tryGetAttachedPropertyByReflection: AssemblyDefinition array -> string * string -> Models.ReflectionAttachedProperty option)
```

Comment:
System.Reflection does not work with Mono.Android (I do not know why)

Reply:
Here, the Reflection is only used at build time by Fabulous.CodeGen. So it won't be used on the Android device at runtime.